crates/vault-registry/src/types.rs

```diff
@@ -238,6 +393,8 @@ pub struct Vault<AccountId, BlockNumber, Balance, CurrencyId: Copy> {
     /// Block height until which this Vault is banned from being used for
     /// Issue, Redeem (except during automatic liquidation) and Replace.
     pub banned_until: Option<BlockNumber>,
+    /// Custom secure collateral threshold
+    pub secure_collateral_threshold: Option<UnsignedFixedPoint>,
```

Comment:
Is there a particular reason this is not inserted at the end of the struct?

Reply:
Not really, but is there a particular reason it should be? Happy to move it, I'm not aware of it making a difference